test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -383,6 +383,352 @@ class CharacterTest {
+    assertEquals(0x1fb7, Character.toUpperCase(0x1fb7)) // ᾷ => ᾷ
+    assertEquals(0x1fbc, Character.toUpperCase(0x1fbc)) // ᾼ => ᾼ
+    assertEquals(0x1fc2, Character.toUpperCase(0x1fc2)) // ῂ => ῂ
+    assertEquals(0x1fcc, Character.toUpperCase(0x1fc3)) // ῃ => ῌ
+    assertEquals(0x1fc4, Character.toUpperCase(0x1fc4)) // ῄ => ῄ
+    assertEquals(0x1fc6, Character.toUpperCase(0x1fc6)) // ῆ => ῆ
+    assertEquals(0x1fc7, Character.toUpperCase(0x1fc7)) // ῇ => ῇ
+    assertEquals(0x1fcc, Character.toUpperCase(0x1fcc)) // ῌ => ῌ
+    assertEquals(0x1fd2, Character.toUpperCase(0x1fd2)) // ῒ => ῒ
+    assertEquals(0x1fd3, Character.toUpperCase(0x1fd3)) // ΐ => ΐ
+    assertEquals(0x1fd6, Character.toUpperCase(0x1fd6)) // ῖ => ῖ
+    assertEquals(0x1fd7, Character.toUpperCase(0x1fd7)) // ῗ => ῗ
+    assertEquals(0x1fe2, Character.toUpperCase(0x1fe2)) // ῢ => ῢ
+    assertEquals(0x1fe3, Character.toUpperCase(0x1fe3)) // ΰ => ΰ
+    assertEquals(0x1fe4, Character.toUpperCase(0x1fe4)) // ῤ => ῤ
+    assertEquals(0x1fe6, Character.toUpperCase(0x1fe6)) // ῦ => ῦ
+    assertEquals(0x1fe7, Character.toUpperCase(0x1fe7)) // ῧ => ῧ
+    assertEquals(0x1ff2, Character.toUpperCase(0x1ff2)) // ῲ => ῲ
+    assertEquals(0x1ffc, Character.toUpperCase(0x1ff3)) // ῳ => ῼ
+    assertEquals(0x1ff4, Character.toUpperCase(0x1ff4)) // ῴ => ῴ
+    assertEquals(0x1ff6, Character.toUpperCase(0x1ff6)) // ῶ => ῶ
+    assertEquals(0x1ff7, Character.toUpperCase(0x1ff7)) // ῷ => ῷ
+    assertEquals(0x1ffc, Character.toUpperCase(0x1ffc)) // ῼ => ῼ
+    assertEquals(0xfb00, Character.toUpperCase(0xfb00)) // ﬀ => ﬀ
+    assertEquals(0xfb01, Character.toUpperCase(0xfb01)) // ﬁ => ﬁ
+    assertEquals(0xfb02, Character.toUpperCase(0xfb02)) // ﬂ => ﬂ
+    assertEquals(0xfb03, Character.toUpperCase(0xfb03)) // ﬃ => ﬃ
+    assertEquals(0xfb04, Character.toUpperCase(0xfb04)) // ﬄ => ﬄ
+    assertEquals(0xfb05, Character.toUpperCase(0xfb05)) // ﬅ => ﬅ
+    assertEquals(0xfb06, Character.toUpperCase(0xfb06)) // ﬆ => ﬆ
+    assertEquals(0xfb13, Character.toUpperCase(0xfb13)) // ﬓ => ﬓ
+    assertEquals(0xfb14, Character.toUpperCase(0xfb14)) // ﬔ => ﬔ
+    assertEquals(0xfb15, Character.toUpperCase(0xfb15)) // ﬕ => ﬕ
+    assertEquals(0xfb16, Character.toUpperCase(0xfb16)) // ﬖ => ﬖ
+    assertEquals(0xfb17, Character.toUpperCase(0xfb17)) // ﬗ => ﬗ
+  }
+
+/*
+def format(codePoint: Int): String = "0x%04x".format(codePoint)
+
+val res = for {
+  cp <- 0 until Character.MAX_CODE_POINT
+  chrUpperCase: Int = Character.toUpperCase(cp)
+  chrUpperCaseStr: String = new String(Array(chrUpperCase), 0, 1)
+  if chrUpperCaseStr.length > 1
+} yield {
+  //val strUpperCase: String = new String(Array(cp), 0, 1).toUpperCase()
+  s"    assertEquals(${format(chrUpperCase)}, Character.toUpperCase(${format(cp)}))"
+}
+
+val r = new scala.util.Random(100)
+r.shuffle(res).take(100).foreach(println)
+*/
+  @Test def toUpperCase_Int_MultiCodePoints(): Unit = {
+    // 100 "Random" codepoints of the 1Mil+
```

Comment:
To be honest, I don't think this is very useful. It would be useful if we implemented the general uppercasing ourselves. But in fact we delegate most of the work to JavaScript's uppercasing.

So what's important to test is:
* Any value that is a special case in our algorithms (:+1: for `toUpperCase_CodePoint_special_cases` and `toUpperCase_CodePoint_StringUpperCase_diff_CharacterUpperCase`)
* A few values to exercise the different code paths in our algorithms. For that I believe the list I gave earlier, with one representative code point in each category, is much more useful than this list of random values. Because that way we exercise each code path. And that should be in `toUpperCase_Int`.

The same comment applies to `toLowerCase_Int`.

Other than that, LGTM (notably, the `toTitleCase` commit with its tests LGTM).